tasks/infra/config/validateJMX.go

```diff
@@ -16,22 +15,17 @@ const defaultJMXport = "9999"
 
 // InfraConfigValidateJMX - This struct defines the task
 type InfraConfigValidateJMX struct {
-	mCmdExecutor func(cmdWrapper, cmdWrapper) ([]byte, error)
-}
-
-//cmdWrapper is used to specify commands & args to be passed to the multi-command executor (mCmdExecutor)
-//allowing for: cmd1 args | cmd2 args
-type cmdWrapper struct {
-	cmd  string
-	args []string
+	mCmdExecutor func(tasks.CmdWrapper, tasks.CmdWrapper) ([]byte, error)
 }
 
 type JmxConfig struct {
-	Host            string
-	Port            string
-	User            string
-	Password        string
-	CollectionFiles string
+	Host                  string
+	Port                  string
+	User                  string
+	Password              string
+	CollectionFiles       string
+	JavaVersion           string
+	JmxProcessCmdlineArgs string
```

Comment:
Is it possible for to collect sensitive information in the cmd line args of the jmx process - like auth / keystore password? If so, we may want to flag this task with a y/n prompt, or scrub values from known sensitive args.

Reply:
Yup, that's a good call. I'm removing the argument values that possibly point to sensitive data. Though, after reading the oracle docs, it seems like at most they can pass references to files where they hide that sensitive data, such as `-Dcom.sun.management.jmxremote.password.file=/pathtofilewhereihidemyauthenticationinfo`. But you never know, so I'm still redacting those values.